test/src/main/java/org/apache/accumulo/test/compaction/ExternalCompactionProgressIT.java

```diff
@@ -75,15 +81,16 @@ public void configureMiniCluster(MiniAccumuloConfigImpl cfg, Configuration coreS
 
   @Test
   public void testProgress() throws Exception {
-    MiniAccumuloClusterImpl.ProcessInfo c1 = null, coord = null;
     String table1 = this.getUniqueNames(1)[0];
     try (AccumuloClient client =
         Accumulo.newClient().from(getCluster().getClientProperties()).build()) {
-      ExternalCompactionTestUtils.createTable(client, table1, "cs1");
-      ExternalCompactionTestUtils.writeData(client, table1, ROWS);
-      c1 = ((MiniAccumuloClusterImpl) getCluster()).exec(Compactor.class, "-q", "DCQ1");
-      coord = ExternalCompactionTestUtils.startCoordinator(((MiniAccumuloClusterImpl) getCluster()),
-          CompactionCoordinator.class, getCluster().getServerContext());
+      createTable(client, table1, "cs1");
+      writeData(client, table1, ROWS);
+
+      ((MiniAccumuloClusterImpl) cluster).getClusterControl().startCompactors(Compactor.class, 1,
```

Comment:
Is the casting to `MiniAccumuloClusterImpl` still necessary here, or is it reachable from the IT superclass without that? I don't recall this being done in other tests. It might not be necessary anymore.

Reply:
Added startCoordinator and startCompactors to ClusterControl interface in [851c2c3] to remove casting. Throw UnsupportedOperationException for these methods in StandaloneClusterControl as they are not implemented yet.

Reply:
Cool. I think there were a few other casts like this, including the commented out one.

Reply:
I saw the one that was in comments. Fixed in 47981eb. Some of the other casts required more work to add the missing methods to the ClusterControl interface.